Sources/MongoSwift/MongoCollection+Indexes.swift

```diff
@@ -339,12 +335,14 @@ extension MongoCollection {
      */
     @discardableResult
     public func dropIndexes(options: DropIndexOptions? = nil) throws -> Document {
-        return try _dropIndexes(options: options)
+        return try _dropIndexes(index: "*", options: options)
     }
 
-    private func _dropIndexes(keys: Document? = nil, options: DropIndexOptions? = nil) throws -> Document {
+    /// Internal helper to drop an index. `index` must either be an index specification document or a
+    /// string index name.
+    private func _dropIndexes(index: BSONValue, options: DropIndexOptions? = nil) throws -> Document {
```

Comment:
I considered trying to make the typing of `index` here a bit more strict to only allow string or document but didn't think it was really worth adding a new protocol or something.